packages/aws-cdk-lib/aws-codepipeline/README.md

```diff
@@ -365,6 +365,8 @@ const replicationBucket = new s3.Bucket(replicationStack, 'ReplicationBucket', {
 
 ## Variables
 
+### Action-level variables
+
```

Comment:
Can we go over the existing README for variables and make sure that they are clear about it being action-level?